### ISSUE TITLE
Release pipeline fails to build docker images

### ISSUE THREAD

**Issue by vanneback** [elastisys/ck8s-cluster#15]
The release pipeline fail everytime on build and push dockerfiles. e.g. https://github.com/elastisys/ck8s-cluster-old/actions/runs/248111401

This problem is because the action want a USERNAME PASSWORD set in secrets so it has credentials for pushing to dockerhub but those are not added in this repo.

**Comment by simonklb** [comment]
This is intentional. If we want to setup automatic build and push let's do it without introducing secrets in this github repo.

**Comment by vanneback** [comment]
Those secrets are still not accessible to others? anyway i think we should reintroduce this since its a good way to have reproducible environments for old/current releases

**Comment by simonklb** [comment]
Absolutely! I'm all for utilizing images instead of installing dependencies on the host.
